gdo_voicebot/dialog_manager_service/config/config.docker.yml

```diff
@@ -27,7 +27,8 @@ policies:
 
 # Configuration for Graphql
 graphql:
-  - url: http://localhost:4000/graphql
+  #- url: http://localhost:4000/graphql
+  - url: http://gdo-students.dsi.ic.ac.uk:4000/graphql
```

Comment:
Since this is an example, can you please replace the gdo-students.dsi.ic.ac.uk with localhost or "specify-the-gql-server-ip-here"